src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.v2.js

```diff
@@ -42,7 +48,7 @@ export function transformFormToVAOSCCRequest(state) {
     status: 'proposed',
     locationId: data.communityCareSystemId,
     // This may need to change when we get the new service type ids
-    serviceType: typeOfCare.ccId,
+    serviceType: typeOfCare.id,
```

Comment:
On a scrum of scrums we decided that we're sending the same types of care between VA and CC. We were handling this correctly in the appointment transform, but not sending the right value here.